content-src/components/LinkMenu/LinkMenu.js

```diff
@@ -12,13 +12,15 @@ const LinkMenu = React.createClass({
     };
   },
   userEvent(event) {
-    const {page, source, index, dispatch} = this.props;
+    const {page, source, index, dispatch, site} = this.props;
     if (page && source) {
       let payload = {
         event,
         page: page,
         source: source,
-        action_position: index
+        action_position: index,
+        url: site.recommended ? site.url : null,
+        recommender_type: site.recommended ? site.recommender_type : null
```

Comment:
Do we need to do any checks that `site` is defined? Or are we guaranteed that we'll have a `site` object if `page` and `source` are truthy?